hugo_build.py

```diff
@@ -1,20 +1,47 @@
 import os
 import shutil
 
+import datetime
 
-def main():
-    registry_dir = "optunahub-registry/package"
+import frontmatter
+
+from git import Repo
+
+
+def main() -> None:
+    repo = Repo("optunahub-registry")
+    tree = repo.tree()
+
+    package_dir = "package"
+    registry_dir = f"optunahub-registry/{package_dir}"
     categories = ["samplers", "pruners", "visualization"]
     for c in categories:
         packages = os.listdir(f"{registry_dir}/{c}")
         for p in packages:
             hugo_dir = f"content/{c}/{p}"
+
+            # Copy README.md and images
             os.makedirs(hugo_dir, exist_ok=True)
-            shutil.copy(f"{registry_dir}/{c}/{p}/README.md", f"{hugo_dir}/index.md")
             imgdir = f"{registry_dir}/{c}/{p}/images"
             if os.path.exists(imgdir):
                 shutil.copytree(imgdir, f"{hugo_dir}/images", dirs_exist_ok=True)
 
+            # Read README.md and create index.md from it with modifications
+            with open(f"{registry_dir}/{c}/{p}/README.md", "r") as readme_md:
+                contents = frontmatter.load(readme_md)
+                with open(f"{hugo_dir}/index.md", "w") as index_md:
+                    # Last update for the package
+                    updated_at_epoch = next(
+                        repo.iter_commits(
+                            paths=tree[f"{package_dir}/{c}/{p}"].path, max_count=1
+                        )
+                    ).committed_date
+                    updated_at_iso = datetime.datetime.fromtimestamp(
+                        updated_at_epoch
+                    ).isoformat()
```

Comment:
As can be seen on the top page, how about showing the date?
The current implementation might be too specific, and users might get confused due to the difference in time zones.

```suggestion
                    updated_at_iso = datetime.datetime.fromtimestamp(
                        updated_at_epoch
                    ).date().isoformat()
```

Reply:
Information on not only the date but also the time is used to sort packages properly.

So, instead of removing the time information, I want to apply formatting to display only the date information in `single.html`.

<img width="258" alt="image" src="https://github.com/user-attachments/assets/2d8e8496-ebdb-4d91-a57b-8a3c0d05080a">

Reply:
Thanks. That must be better.